Translation-Italian.md

```diff
@@ -0,0 +1,102 @@
+[English](./readme.md) ·
+[中文](./Translation-Chinese.md) ·
+[Español](./Translation-Spanish.md) ·
+[Português](./Translation-Portuguese.md) ·
+[Français](./Translation-French.md) ·
+[Русский](./Translation-Russian.md) ·
+[Deutsch](./Translation-German.md) ·
+[日本語](./Translation-Japanese.md) .
+Italiano .
+
+---
+
+# LaTeX in pochi minuti
+
+![](https://upload.wikimedia.org/wikipedia/commons/9/92/LaTeX_logo.svg)
+
+**Riconoscimenti:** *Quanto scritto in questo documento deriva da mie esperienze universitarie e letture di diverso materiale sull'argomento. Non sono un professionista o un esperto ma un semplice studente con una passione per l'argomento. Chiunque può aprire una discussione nella sezione "Problemi" o aprire una Pull Request nel caso ritenga che qualcosa debba essere aggiunto o modificato. Se il mio lavoro vi è tornato utile una [donazione](#donation) è molto apprezzata.*
+
+### Sommario
+
+* [Che cos'è LaTeX?](#che-cos'è-latex)
```

Comment:
This link does not work.

Reply:
It's actually working on my machine, see attached gif
![Latex_links](https://user-images.githubusercontent.com/15200602/60093805-a4d04f00-974a-11e9-8ff1-09e1f6cd2b12.gif)

Reply:
Unfortunately, it does not matter. It should work on GitHub.

Here is [your link](https://github.com/Kastakin/Begin-Latex-in-minutes/blob/translate/italian/Translation-Italian.md#che-cos'è-latex). Here is the [link that works](https://github.com/Kastakin/Begin-Latex-in-minutes/blob/translate/italian/Translation-Italian.md#che-cosè-latex).

Reply:
Gonna check on it!